infra/modules/worklytics-connector-specs/main.tf

```diff
@@ -274,15 +275,15 @@ EOT
       target_host : "api.github.com"
       source_auth_strategy : "oauth2_refresh_token"
       secured_variables : [
-        { name : "ACCESS_TOKEN", writable : true },
+        { name : "PRIVATE_KEY", writable : true },
```

Comment:
Very similar with client_credentials for MSFT certificates; here should be the value of the converted PKSC8 certificate downloaded from the application created.

Reply:
we have to be super clear; may be worth adding 'description' field here as well, so it even ends up being specified on the parameter/secret what this value is and where it's coming from, particularly where it depends on a specific format.  

with description, person seeing it in AWS / GCP console when they fill it might notice if they're about to fill the wrong format (prob not super relevant here, as most people won't know whether they have PKSC8 or not)